.gitattributes

```diff
@@ -1,9 +1,32 @@
-LICENSE text
-*.md text
-*.js text
-*.json text
-*.html text
-*.css text
-*.txt text
+LICENSE text eol=lf
+*.md text eol=lf
+*.c text eol=lf
+*.cc text eol=lf
+*.h text eol=lf
+*.cc text eol=lf
+*.S text eol=lf
+*.s text eol=lf
+*.hex -crlf -diff
+*.elf -crlf -diff
+*.ld text eol=lf
+Makefile text eol=lf
+*.mk text eol=lf
+*.nomk text eol=lf
+*.pl text eol=lf
+*.php text eol=lf
```

Comment:
php? eeek.

Reply:
Probably shoulda delete that one instead of recording it for posterity:

https://github.com/betaflight/betaflight/commits/master/support/buildserver/upload.php

Reply:
betaflight/betaflight#8975